client/modules/workspace/src/AppsSubmissionForm/AppsSubmissionForm.tsx

```diff
@@ -615,23 +620,47 @@ export const AppsSubmissionForm: React.FC = () => {
           >
             <fieldset disabled={readOnly}>
               <Row gutter={[64, 16]} align="top">
-                <Col span={14}>
-                  <AppsWizard
-                    step={steps[current]}
-                    handlePreviousStep={handlePreviousStep}
-                    handleNextStep={handleNextStep}
-                  />
-                </Col>
-                <Col span={10}>
-                  <AppsSubmissionDetails
-                    schema={schema}
-                    fields={fields}
-                    isSubmitting={isPending}
-                    current={current}
-                    setCurrent={setCurrent}
-                    definition={definition}
-                  />
-                </Col>
+                {Object.keys(steps || {}).length === 0 ? (
+                  <Col style={{ marginTop: '32px', marginLeft: '32px' }}>
+                    <Space direction="vertical" size="large">
+                      <div>
+                        {definition.notes.jobLaunchDescription ??
+                          'This job is pre-configured. No input is necessary to submit the job.'}
```

Comment:
@wesleyboar I wonder, should we have any post-launch text change or state change? i.e. make the button disabled and add a check mark indicating the session was launched. Might be good to note if the modal is closed.

Reply:
@chandra-tacc Maybe we can use the same pattern here: https://github.com/DesignSafe-CI/portal/blob/e409cbe9bc0b2b7c033d08b22e681f09b78e9dbd/client/modules/workspace/src/JobsListing/JobsListing.tsx#L51

Reply:
Great idea @rstijerina. Made that change. Below are screenshots that show in various states.

### Launched with modal open. Launch is disabled.
![Screenshot 2024-10-10 at 3 04 51 PM](https://github.com/user-attachments/assets/dc34f18d-d1d6-4d38-91f8-49ec55f3ed14)


### Launch with modal closed. 
![Screenshot 2024-10-10 at 3 05 02 PM](https://github.com/user-attachments/assets/60b014d9-4af2-4f3d-8541-4d172a45499f)

### Launch after page refresh, resets to enabled.
![Screenshot 2024-10-10 at 3 07 52 PM](https://github.com/user-attachments/assets/2492798a-ac82-4a12-a56b-0e2021a5efd2)

Reply:
Looks great! Maybe also include the check mark?
https://github.com/DesignSafe-CI/portal/blob/e409cbe9bc0b2b7c033d08b22e681f09b78e9dbd/client/modules/workspace/src/JobsListing/JobsListing.tsx#L56

Reply:
There is more work, will need to make a bit more adjustment based on success state.

Reply:
@wesleyboar Should we also change the text once disabled, to:
"Session has been launched. You can view status in <a>Job Status</a>"

Reply:
Yeah, if we have the state, lets. Great idea.

Reply:
@rstijerina  - done made the adjustments.
1. Button text has the check mark on job success.
2. Job success text changes based on state.

Screen recording of this:

https://github.com/user-attachments/assets/5ac47ccb-d412-4430-814a-96105ee137b7